boards/shields/lcd_par_s035/boards/frdm_mcxn947_mcxn947_cpu0.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&gt911_lcd_par_s035 {
+	irq-gpios = <&gpio4 6 GPIO_ACTIVE_HIGH>;
```

Comment:
We should not put this in a board overlay- instead, we should use a gpio nexus node, with a compatible appropriate for the connector we are describing (perhaps `nxp_mipi_dbi_gpio`?). For an example, see here: https://github.com/zephyrproject-rtos/zephyr/pull/72174

Reply:
Hi @danieldegrasse , I guess I should add a yaml for LCD-PAR-S035 connecter (including both the 8080 pin and I2C pin), is my  understanding right?
Another question is, this LCD-PAR-S035 connecter is not a standard connecter, the name `nxp_mipi_dbi_gpio` might be too generic, is there some suggestion for the naming? thanks.

Reply:
The YAML only needs to define the GPIO pins not used directly by a peripheral- typically interrupt/reset pins are needed there. We do need a YAML for the LCD-PAR connector though. Not sure how best to name this- do we expect the LCD_PAR to be the only NXP display that uses this connector?